apps/web/src/components/activity/ExecutionDetailShowRaw.tsx

```diff
@@ -94,7 +93,7 @@ export const ExecutionDetailRawSnippet = ({ raw, onClose }) => {
         data-test-id="execution-detail-raw-snippet"
         language="json"
       >
-        {raw}
+        {JSON.stringify(JSON.parse(raw), null, 2)}
```

Comment:
I tried this but I wasn't able to work. Thanks for fixing it!

Reply:
No problem took me a while to understand how to resolve this 👍